deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvironBean.java

```diff
@@ -59,8 +59,9 @@
  * cluster_name        VARCHAR(128)
  * max_parallel_rp     INT           NOT NULL DEFAULT 1,
  * schedule_id         VARCHAR(22)   DEFAULT NULL,
- * deploy_constraint_id VARCHAR(22)   DEFAULT NULL
- * external_id CHAR(36)
+ * deploy_constraint_id VARCHAR(22)   DEFAULT NULL,
+ * external_id CHAR(36),
+ * allow_private_build TINYINT(1)    DEFAULT 0
  * <p>
```

Comment:
What happen if a new whitelist stage came? Overwrite previous whitelist stage?

Reply:
when a stage is created, the allow_private_build column/flag is default to false, meaning that no private build is allowed.  Customer need to request Teletraan Admin to whitelist this stage by setting the column/flag to 1.

Reply:
Thanks for explaining. What is next step?  Once Teletraan Admin set flag to 1 in DB for them, customers can use "public private build" button to upload their build, right?